src/scales/opacityScale.ts

```diff
@@ -0,0 +1,56 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Scales {
+  export class Opacity extends Scale<string, number> {
```

Comment:
I think there's probably a more generalized way to build this. Essentially, this `Scale` maps one-to-one from `String`s to `number`s, but that functionality has applications beyond setting an opacity -- the `number` could have other meanings.

In a more general sense, it may be helpful to have the `Scale` map one-to-one with other types of values, such as `String`s (which would take care of #746), so it might be helpful to leave the *range* generic.